styles/base.css

```diff
@@ -42,40 +54,106 @@ a {
     transition: 250ms;
 }
 
-a:hover, a:focus {
+a:hover,
+a:focus {
     color: var(--green);
 }
 
 ul.no-bullets {
+    padding-left: 0;
+}
+
+ul.no-bullets li {
     list-style-type: none;
     padding: 0;
-    margin: inherit 0 0;
+    margin: 16px 0;
+}
+
+@media (min-width: 992px) {
+    ul.no-bullets li {
+        margin: inherit 0 0;
+    }
 }
 
 li.certs {
     padding-bottom: 0.75rem;
 }
 
-.monospace, h1, h2, h3, h4, h5, h6 {
+/*#region Font Family */
+
+.monospace,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
     font-family: 'Source Code Pro Bold', 'Courier New', Courier, monospace;
 }
 
 .san-serif {
     font-family: 'Roboto', Arial, Helvetica, sans-serif;
 }
 
-.title-lg {
+/*#endregion */
+
+/*#region Font Sizes */
+
+:root {
+    --title-font-size: 1.75rem;
+    --button-text-font-size: 1.25rem;
+}
+
+@media (min-width: 992px) {
+    :root {
+        --title-font-size: 2rem;
+        --button-text-font-size: 1.5rem;
+    }
+}
+
+.lg-link {
+    font-size: var(--title-font-size);
+}
+
+.btn-link {
+    font-size: var(--button-text-font-size);
+}
+
+/*
+Overwrite Bootstrap's heading font-sizes
+https://stackoverflow.com/questions/5410066/what-are-the-default-font-sizes-in-pixels-for-the-html-heading-tags-h1-h2
+*/
+
+h1 {
     font-size: 2rem;
+    margin-top: 1.5rem;
+    margin-bottom: 1.5rem;
 }
 
-.text-lg {
+h2 {
     font-size: 1.5rem;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+}
+
+h3 {
+    font-size: 1.17rem;
+}
+
+h4 {
+    font-size: 1rem;
 }
 
-.shifted-down {
-    margin-top: 50px;
+h5 {
+    font-size: 0.83rem;
 }
 
+h6 {
+    font-size: 0.67rem;
+}
+
```

Comment:
I think(?!?) these h3-h6 styles can be removed cos the app is not using any yet.